docs/01-app/02-building-your-application/04-caching/index.mdx

```diff
@@ -434,6 +440,22 @@ You can opt individual `fetch` into caching by setting the `cache` option to `fo
 fetch(`https://...`, { cache: 'force-cache' })
 ```
 
+Alternatively you can set the global caching behavior in your `next.config.js` file as follows::
+
+```ts filename="next.config.ts" switcher
+import type { NextConfig } from 'next'
+
+const nextConfig = {
+  experimental: {
+    staleTimes: {
+      dynamic: 30,
+    },
+  },
+};
+
+export default nextConfig
+```
```

Comment:
I think this might confuse some people if they're not aware `staleTimes` affects the client-side router cache, and not the server-side router and data cache. I suggest adding more nuance as to why you'd reach out for this config option or removing it altogether. wdyt?